share/commands/server-directive-migrate-package

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+# @description create a new package directive based on an out-to-date package directive.
+# @man create a new package directive based on an out-to-date package directive.
+# @man +
+# @man Out-to-date package techniques are {"aptPackageInstallation", "rpmPackageInstallation"}.
+# @man Not all of the options present in the old package techniques are still available.
+# @man The script will stop and do nothing if one parameter is not translatable.
+# @man +
+# @man If the script succeed, it will create a new directive, without rule assignement
+# @man and the old directive will not be changed to ensure human verification on the
+# @man parameters translation.
+# @man +
+# @man *Arguments*:
+# @man +
+# @man *-o*: old directive id
+# @man +
+# @man *-[c]*: raw color mode, all output are non-colored text
```

Comment:
we don't use [] for optional parameters, this usually belongs to the synopsis which we currently don't have